simple-calc/modules/MVUX-CSharp/04-App Architecture/README.md

```diff
@@ -4,12 +4,12 @@ uid: Workshop.SimpleCalc.MVUX.CSharp.Architecture
 
 [!include[Introduction](../../Resources/MVUX/Intro.md)]
 
-To start we need to replace the temporary DataContext class we created earlier. It's also important to note here that we will not initialize the MainModel since we need the `X` in `MVUX`. For this, we will use the generated `BindableMainModel` class. You can do a Find/Replace to replace all references to `TempDataContext` with `BindableMainModel`, and then remove the `TempDataContext` class.
+To start we need to replace the temporary DataContext class we created earlier. It's also important to note here that we will not initialize the MainModel since we need the `X` in `MVUX`. For this, we will use the generated `MainViewModel` class. You can do a Find/Replace to replace all references to `TempDataContext` with `MainViewModel`, and then remove the `TempDataContext` class.
```

Comment:
@eriklimakc, @jeromelaban based on what I shared for the samples [here](https://github.com/unoplatform/Uno.Samples/pull/818#discussion_r1733474955), these changes for the workshops documentation will need to be merged and released only when a new stable version of uno.sdk will include these changes right?